chromadb/utils/embedding_functions.py

```diff
@@ -148,9 +148,9 @@ def __init__(
 
     def __call__(self, texts: Documents) -> Embeddings:
         # Call HuggingFace Embedding API for each document
-        return self._session.post(  # type: ignore
+        return [self._session.post(  # type: ignore
             self._api_url, json={"inputs": texts, "options": {"wait_for_model": True}}
-        ).json()
+        ).json()]
```

Comment:
I tested this and this isn't right. I'm going to take over the PR, remove this and land